ERROR: Runtime error while executing "M".
at model "M" near line 14.
CAUSE: Cannot find a biggest value in an empty set.
